google/resumable_media/_download.py

```diff
@@ -58,6 +59,8 @@ def __init__(self, media_url, stream=None, start=None, end=None, headers=None):
         self.end = end
         if headers is None:
             headers = {}
+        version = pkg_resources.get_distribution('google-resumable-media').version
+        headers['User-Agent'] = "google-resumable-media-python/{}".format(version)
```

Comment:
Right now we internally track `google-python/{}`; we might need to update this to `google-python-resumable-media/{}`, my only queestion is if we need to create another UserAgent instead of just using `google-python/{}`

Reply:
That sounds reasonable. @andrewsg any thoughts?

Reply:
I think google-python might actually make the most sense, since this is mostly called from inside of python-storage and so giving it it's own user agent could be confusing.

Reply:
I did see that https://github.com/googleapis/google-resumable-media-python/pull/84#issuecomment-525827813, the transport passed into this package is expected to have the user-agent. I'm thinking we may want to make this change within python-storage only; however we would lose tracking for this package if used outside of python-storage. 

I raised the question in Yoshi Chat room because I'm a bit confused.

Reply:
Ha, andrew and I are sharing with you in much of that confusion. Perhaps we can all hop on a call and discuss.

Reply:
python-storage uses gcloud-python, not google-python. Both this library and python-storage include version numbers, and if we use the same header those version numbers will conflict. I'll continue discussion in chat.